config/formats.ts

```diff
@@ -1,8 +1,81 @@
 // Note: This is the list of formats
-// The rules that formats use are stored in data/rulesets.js
+// The rules that formats use are stored in data/rulesets.ts
 import {Utils} from './../lib/utils';
 
-export const Formats: (FormatsData | {section: string, column?: number})[] = [
+import { SideFormats, FormatList } from './custom-formats';
```

Comment:
```suggestion
import {SideFormats, FormatList} from './custom-formats';
```

Reply:
someone suggested this change.... Oh well.

Reply:
Someone suggested I add the spaces. Oh well.